node/src/accountant/db_access_objects/sent_payable_dao.rs

```diff
@@ -20,37 +20,33 @@ pub enum SentPayableDaoError {
     SqlExecutionFailed(String),
 }
 
-type TxHash = H256;
+type Hash = H256;
 type RowId = u64;
 
-type TxIdentifiers = HashMap<TxHash, RowId>;
-type TxUpdates = HashMap<TxHash, TxStatus>;
+type TxIdentifiers = HashMap<Hash, RowId>;
+type TxUpdates = HashMap<Hash, TransactionBlock>;
 
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct Tx {
-    pub hash: TxHash,
+    pub hash: Hash,
```

Comment:
I've gone through this PR and I kinda grew fond of `TxHash`. Not sure why you gave that up. Was it me suggesting that?

Reply:
It's because it can now be used to represent either Block's hash or transaction's hash, that's why I made this change to make it generic.